examples/routers/index.js

```diff
@@ -21,6 +21,7 @@ import Popover from './popover'
 import Popconfirm from './popconfirm'
 import Timeline from './timeline'
 import Dropdown from './dropdown'
+import Other from './other'
```

Comment:
why named it 'other'?

Reply:
in antd .it is other.backTop
https://ant.design/docs/react/introduce-cn

Reply:
read it again.